examples/chatterbox.py

```diff
@@ -0,0 +1,303 @@
+Vec3 = require("vec3").Vec3
+
+print(sys.argv)
+if len(sys.argv) < 3 or len(sys.argv) > 6:
+    console.log("Usage : node chatterbot.js <host> <port> [<name>]")
+    exit(1)
+
+host = sys.argv[1]
+port = sys.argv[2]
+username = sys.argv[3] if len(sys.argv) > 3 else "boat"
+
+bot = mineflayer.createBot({
+    "host": host,
+    "port": port,
+    "username": username,
+    "port": port
+})
+
+Item = require("prismarine-item")(bot.version)
+
+
+@On(bot, "chat")
+def handle(this, username, message, *args):
+    if username == bot.username:
+        return
+
+    if message.startswith("can see"):
+        # Extract x, y and z
+        # e.g. "can see 327 60 -120" or "can see 327, -23, -120"
+        try:
+            x, y, z = map(lambda v: int(v), message.split("see")[1].replace(",", " ").split())
+        except Exception:
+            bot.chat("Bad syntax")
+    elif message.startswith("pos"):
+        say_position(username)
+    elif message.startswith("wearing"):
+        say_equipment(username)
+    elif message.startswith("block"):
+        say_block_under()
+    elif message.startswith("spawn"):
+        say_spawn()
+    elif message.startswith("quit"):
+        quit_game(username)
+    else:
+        bot.chat("That's nice")
+
+
+def can_see(pos):
+    block = bot.blockAt(pos)
+    canSee = bot.canSeeBlock(block)
+
+    if canSee:
+        bot.chat(f"I can see the block of {block.displayName} at {pos}")
+    else:
+        bot.chat(f"I can't see the block of {block.displayName} at {pos}")
+
+
+def say_position(username):
+    p = bot.entity.position
+    bot.chat(f"I am at {p.toString()}")
+    if username in bot.players:
+        p = bot.players[username].entity.position
+        bot.chat(f"You are at {p.toString()}")
+
+
+def say_equipment(username):
+    eq = bot.players[username].entity.equipment
+    eqText = []
+    if eq[0]:
+        eqText.append(f"holding a {eq[0].displayName}")
+    if eq[1]:
+        eqText.append(f"wearing a {eq[1].displayName} on your feet")
+    if eq[2]:
+        eqText.append(f"wearing a {eq[2].displayName} on your legs")
+    if eq[3]:
+        eqText.append(f"wearing a {eq[3].displayName} on your torso")
+    if eq[4]:
+        eqText.append(f"wearing a {eq[4].displayName} on your head")
+    if len(eqText):
+        bot.chat(f"You are {', '.join(eqText)}.")
+    else:
+        bot.chat("You are naked!")
+
+
+def say_spawn():
+    bot.chat(f"Spawn is at {bot.spawnPoint.toString()}")
+
+
+def say_block_under():
+    block = bot.blockAt(bot.players[username].entity.position.offset(0, -1, 0))
+    bot.chat(f"Block under you is {block.displayName} in the {block.biome.name} biome")
+    print(block)
+
+
+def quit_game(username):
+    bot.quit(f"{username} told me to")
+
+
+def say_nick():
+    bot.chat(f"My name is {bot.player.displayName}")
+
+
+@On(bot, "whisper")
+def whisper(this, username, message, rawMessage, *a):
+    console.log(f"I received a message from {username}: {message}")
```

Comment:
Shouldn't the console logs be print statements?

Reply:
no, console is imported at the top

Reply:
It works the same way, but I've fixed it to use `print`.